setup.py

```diff
@@ -64,7 +64,7 @@
     python_requires='>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*',
     install_requires=install_requires,
     extras_require={
-        "plotting":  ['pyvista>=0.24.0', 'matplotlib==3.2'],
```

Comment:
Hi @PipKat, why are we changing this version? Changing for < rather then >= means that we have to think about updating it regularly

Reply:
That's my doing.  It's related to our doc failures as commented https://github.com/pyansys/DPF-Core/pull/71#issuecomment-924973090.

There are four tests that will fail unless we cap at `pyvista<0.32` or modify the tests.  It's up to you if you want to modify the tests to get our CI/CD to pass or cap the version, but one will have to be done.

Reply:
@akaszynski @cbellot000 @PipKat I think this is better to update the tests (done in the following PR: #73), and keep the pyvista requirement versioning as it is right now.

Reply:
I've reverted the changes.  We're going to have to wait for #73 to be merged to validate the CI.  I'll let you merge it (never liked it when other people merged my PRs :)